### PR TITLE
rpc: Rename first named arg of createrawtransaction

### LINKED ISSUE #10079: [rpc] Createrawtransaction doc does not match named args
According to the help text of createrawtransaction, the following args are accepted:
```
Arguments:
1. "inputs"                (string, required) A json array of json objects
2. "outputs"               (string, required) a json object with outputs
3. locktime                  (numeric, optional, default=0) Raw locktime. Non-0 value also locktime-activates inputs
```

however, this does not match the named args ( https://github.com/bitcoin/bitcoin/blob/8152d3fe57a991e9088d0b9d261d2b10936f45a9/src/rpc/client.cpp#L84 )

Imo, backporting a breaking change to 0.14.1 is reasonable. Though, I'd like to hear other opinions.

### PR DESCRIPTION
**Pull request by maflcko**
Fixes #10079.

### CODE REVIEW

File: src/rpc/rawtransaction.cpp [hunk 1]
```diff
@@ -369,7 +369,7 @@ UniValue createrawtransaction(const JSONRPCRequest& request)
             "       } \n"
             "       ,...\n"
             "     ]\n"
-            "2. \"outputs\"               (string, required) a json object with outputs\n"
+            "2. \"outputs\"               (object, required) a json object with outputs\n"
```
Comment (dcousens) [inline src/rpc/rawtransaction.cpp]: ACK this fix

**Review by dcousens — APPROVED** [review]
ACK

**Comment by NicolasDorier** [comment]
utACK

**Comment by laanwj** [comment]
utACK. Makes sense. As this is an API change, this needs mention in the release notes.

**Comment by jonasschnelli** [comment]
ACK fa558532192ca0bb519f811ee14df6037413b89f

**Comment by TheBlueMatt** [comment]
Hmm, I suppose it's unlikely anyone is even using named args in 0.14 yet, so probably fine, but as folks start using this we should prefer to update docs over code.

utACK fa558532192ca0bb519f811ee14df6037413b89f

**Comment by laanwj** [comment]
> Hmm, I suppose it's unlikely anyone is even using named args in 0.14 yet, so probably fine, but as folks start using this we should prefer to update docs over code.

In general you are right, but in this specific case 'inputs' is a much better name for the argument. I think this slipped by in review of #8811.

**Pull request merged**